File: Python_Ruby/Function/4.py
input_id = input("Put your ID\n")

def login_id(_id):
    login_memeber = ['naver', 'google', 'duckduckgo']
    for login_members in login_memeber:
        if login_members == _id:
            return True
    return False   
            
if login_id(input_id):
    print('Hello ' +input_id)
else:
    print('Who are you?')        

File: Python_Ruby/Function/4.rb
puts("Put your ID")
input_id = gets.chomp()
 
def login(_id)
  members = ['naver', 'google', 'duckduckgo']
  for member in members do
      if member == _id
          return true
      end
  end
  return false
end
 
if login(input_id)
  puts('Hello, '+input_id)
else
  puts('Who are you?')
end